simulation/src/test/resources/report.md

```diff
@@ -13,7 +13,7 @@
         fast_503s_then_revert[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=100.0%	client_mean=PT0.120012009S 	server_cpu=PT1H30M0.00000004S	client_received=45000/45000	server_resps=45004	codes={200=45000}
             fast_503s_then_revert[CONCURRENCY_LIMITER_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.120126613S 	server_cpu=PT1H30M0.0000004S	client_received=45000/45000	server_resps=45040	codes={200=45000}
                       fast_503s_then_revert[UNLIMITED_ROUND_ROBIN].txt:	success=100.0%	client_mean=PT0.120126613S 	server_cpu=PT1H30M0.0000004S	client_received=45000/45000	server_resps=45040	codes={200=45000}
-               live_reloading[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=97.2%	client_mean=PT4.8253936S   	server_cpu=PT2H46.79S     	client_received=2500/2500	server_resps=2500	codes={200=2430, 500=70}
+               live_reloading[CONCURRENCY_LIMITER_PIN_UNTIL_ERROR].txt:	success=94.0%	client_mean=PT5.2613496S   	server_cpu=PT1H56M35.39S  	client_received=2500/2500	server_resps=2500	codes={200=2350, 500=150}
```

Comment:
Not sure why this simulation was impacted

Reply:
Looks like we're not preserving pinned state on reload: https://github.com/palantir/dialogue/blob/fo/server-side-node-selection/simulation/src/test/resources/report.md#live_reloadingconcurrency_limiter_pin_until_error

Reply:
🤦 looks like the node selection wrapper is wrecking our equality check